tenable/io/tags.py

```diff
@@ -224,7 +224,7 @@ def create(self, category, value, description=None, category_description=None,
 
             >>> tio.tags.create('00000000-0000-0000-0000-000000000000', 'Madison')
         '''
-        all_permissions = ['ALL', 'CAN_EDIT', 'CAN_SET_PERMISSIONS', 'CAN_USE']
+        all_permissions = ['ALL', 'CAN_EDIT', 'CAN_SET_PERMISSIONS']
```

Comment:
Don't we need CAN_USE?

Reply:
reverted changes in this PR and created PR #340 for bugfix